### PR TITLE
refactor(ui5-multi-combobox): update ACC support with valueState description

### PR DESCRIPTION
**Pull request by ivoplashkov**
Added a new attribute aria-describedby in order to inform the user for the current value state if any.

### CODE REVIEW

File: packages/main/src/MultiComboBox.hbs [hunk 1]
```diff
@@ -36,8 +40,9 @@
 		@focusout="{{_focusout}}"
 		role="combobox"
 		aria-haspopup="listbox"
-		aria-labelledby="{{_id}}-hiddenText-nMore">
-	</input>
```
Comment (vladitasev) [inline packages/main/src/MultiComboBox.hbs]: Where do we close the <input tag now?

File: packages/main/src/MultiComboBox.js [hunk 2]
```diff
@@ -506,7 +513,22 @@ class MultiComboBox extends UI5Element {
 	}
 
 	get nMoreCountText() {
-		return this.i18nBundle.getText(TOKENIZER_ARIA_CONTAIN_SEVERAL_TOKENS, this._getSelectedItems().length);
+		const iTokenCount = this._getSelectedItems().length;
+		let sTokenizerAria = "";
+
+		switch (iTokenCount) {
```
Comment (vladitasev) [inline packages/main/src/MultiComboBox.js]: if (iTokenCount === 0) {
 return ...;
}

if (iTokenCount === 1) {
 return .....;
}

return ....;

File: packages/main/src/MultiComboBox.js [hunk 3]
```diff
@@ -531,6 +553,22 @@ class MultiComboBox extends UI5Element {
 		return this.readonly ? "None" : "MultiSelect";
 	}
 
+	valueStateTextMappings() {
```
Comment (vladitasev) [inline packages/main/src/MultiComboBox.js]: This should be a getter, not a function

**Comment by elenastoyanovaa** [comment]
Everything is correct on ACC side.

File: packages/main/src/MultiComboBox.hbs (hunk 1 above)
Comment (ivoplashkov) [reply packages/main/src/MultiComboBox.hbs]: The input is self closed at line 47.

**Pull request merged**